kubernetes/samples/scripts/common/utility.sh

```diff
@@ -389,6 +387,9 @@ function createFiles {
       sed -i -e "s|%IMAGE_NAME%|${image}|g" ${domainPropertiesOutput}
     fi
   else
+    # we're in the domain in PV case
+
+    wdtVersion="${WDT_VERSION:-${wdtVersion}}"
```

Comment:
Implementation note:

The changes in this file only affect the PV path, which is not the main thrust of this pull, but I did this for clarity.  

Specifically, I moved the 'wdtVersion' initialization for PV down into the else clause to make it more clear that it's only used on the PV path, plus, instead of setting 'wdtVersion' to WDT_VERSION directly (WDT_VERSION is an undocumented setting), I set wdtVersion to come from the 'wdtVersion' env var if (a) it's already been set and (b) WDT_VERSION has not been set.  (If the wdtVersion env var has been set, then it probably came from the inputs file.)

Phew!